checkov/terraform/checks/resource/gcp/GoogleComputeDefaultServiceAccount.py

```diff
@@ -28,6 +29,10 @@ def scan_resource_conf(self, conf):
             self.evaluated_keys = ['name']
             return CheckResult.PASSED
         self.evaluated_keys = ['service_account/[0]/email', 'name']
+        if 'source_instance_template' in conf.keys() and 'service_account' not in conf.keys():
+            # the absence of service_account from a 'google_compute_instance_from_template' definition does not imply
```

Comment:
@mikeurbanski1 Let me know what you think about the usage of `CheckResult.UNKNOWN` in this case

Reply:
Yeah, this makes sense to me.